rules/eslint/strict/walmart.js

```diff
@@ -3,6 +3,6 @@
 module.exports = {
   rules: {
     // require that all functions are run in strict mode
-    'strict': [2, "never"]
+    'strict': [2, "always"]
```

Comment:
Per our chat, let's choose a default for "all JS", which I think is still `never`.